freemove.js

```diff
@@ -12,20 +12,20 @@ Freemove.prototype.compute = function(keys, angle) {
   var keymap = this.keymap
 
   if (keymap.position) {
-    if (keymap.position[0][0] in keys) this.velocity.position[0] += this.speed
-    if (keymap.position[0][1] in keys) this.velocity.position[0] -= this.speed
-    if (keymap.position[1][0] in keys) this.velocity.position[1] += this.speed
-    if (keymap.position[1][1] in keys) this.velocity.position[1] -= this.speed
+    if (keymap.position[0][0] in keys) this.velocity.position[0] += this.speed.position
+    if (keymap.position[0][1] in keys) this.velocity.position[0] -= this.speed.position
+    if (keymap.position[1][0] in keys) this.velocity.position[1] += this.speed.position
+    if (keymap.position[1][1] in keys) this.velocity.position[1] -= this.speed.position
   }
 
   if (keymap.angle) {
-    if (keymap.angle[0] in keys) this.velocity.angle += this.speed
-    if (keymap.angle[1] in keys) this.velocity.angle -= this.speed
+    if (keymap.angle[0] in keys) this.velocity.angle += this.speed.angle
+    if (keymap.angle[1] in keys) this.velocity.angle -= this.speed.angle
   }
 
   if (keymap.scale) {
-    if (keymap.scale[0] in keys) this.velocity.scale += this.speed*.01
-    if (keymap.scale[1] in keys) this.velocity.scale -= this.speed*.01
+    if (keymap.scale[0] in keys) this.velocity.scale += this.speed.position*.01
```

Comment:
If you're going to break up speed, might as well make a `speed.scale` no? and do the `*.01` that way?